src/script/calling/CallingRepository.ts

```diff
@@ -1272,7 +1309,11 @@ export class CallingRepository {
 
   readonly sendModeratorMute = (conversationId: QualifiedId, participants: Participant[]) => {
     const recipients = this.convertParticipantsToCallingMessageRecepients(participants);
-    this.sendCallingMessage(conversationId, {type: CALL_MESSAGE_TYPE.REMOTE_MUTE}, {nativePush: true, recipients});
+    this.sendCallingMessage(
+      conversationId,
+      {type: CALL_MESSAGE_TYPE.REMOTE_MUTE, data: {targets: recipients}},
```

Comment:
When sending a `REMOTEMUTE` type of calling message, we need to include `targets` field containing the list of clients that should be muted.